kubeflow/examples/prototypes/tf-serving-with-istio.jsonnet

```diff
@@ -0,0 +1,176 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.tf-serving-with-istio
+// @description tf-serving-with-istio
+// @shortDescription tf-serving-with-istio
+// @param name string Name to give to each of the components
+
+local k = import "k.libsonnet";
+
+local namespace = "default";
+local appName = "tf-serving-with-istio";
+local modelBasePath = "gs://kubeflow-models/inception";
+local modelName = "inception";
+local image = "gcr.io/kubeflow-images-public/tf-model-server-cpu:v20180327-995786ec";
+local httpProxyImage = "gcr.io/kubeflow-images-public/tf-model-server-http-proxy:v20180327-995786ec";
+
+local routeRule = {
+  apiVersion: "config.istio.io/v1alpha2",
+  kind: "RouteRule",
+  metadata: {
+    name: appName,
+    namespace: namespace,
+  },
+  spec: {
+    destination: {
+      name: "tf-serving",
+    },
+    precedence: 0,
+    route: [
+      {
+        labels: {
+          version: "v1",
+        },
+      },
+    ],
+  },
+};
+
+local service = {
+  apiVersion: "v1",
+  kind: "Service",
+  metadata: {
+    annotations: {
+      "getambassador.io/config":
+        std.join("\n", [
+          "---",
+          "apiVersion: ambassador/v0",
+          "kind:  Mapping",
+          "name: tfserving-mapping-tf-serving-get",
+          "prefix: /models/tf-serving/",
+          "rewrite: /",
+          "method: GET",
+          "service: tf-serving." + namespace + ":8000",
+          "---",
+          "apiVersion: ambassador/v0",
+          "kind:  Mapping",
+          "name: tfserving-mapping-tf-serving-post",
+          "prefix: /models/tf-serving/",
+          "rewrite: /model/tf-serving:predict",
+          "method: POST",
+          "service: tf-serving." + namespace + ":8000",
+        ]),
+    },
+    labels: {
+      app: appName,
+    },
+    name: appName,
+    namespace: namespace,
+  },
+  spec: {
+    ports: [
+      {
+        name: "grpc-tf-serving",
+        port: 9000,
+        targetPort: 9000,
+      },
+      {
+        name: "http-tf-serving-proxy",
+        port: 8000,
+        targetPort: 8000,
+      },
+    ],
+    selector: {
+      app: appName,
+    },
+    type: "ClusterIP",
+  },
+};
+
+local deployment = {
+  apiVersion: "extensions/v1beta1",
+  kind: "Deployment",
+  metadata: {
+    labels: {
+      app: appName,
+    },
+    name: appName,
+    namespace: namespace,
+  },
+  spec: {
+    template: {
+      metadata: {
```

Comment:
Need this annotation
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/tf-serving/tf-serving.libsonnet#L172

Reply:
Done